electron_app/src/DBManager.js

```diff
@@ -1071,6 +1071,23 @@ const updateIdentityKeyRecord = ({ recipientId, identityKey }) => {
     .update({ identityKey });
 };
 
+/* Pending Event
+----------------------------- */
+const createPendingEvent = params => {
+  return db.table(Table.PENDINGEVENT).insert(params);
+};
+
+const getPendingEvents = () => {
+  return db.select('*').from(Table.PENDINGEVENT);
```

Comment:
Naming `getPendingEvents` is confusing because the API method that gets inbound peer events is already called `getPendingEvents`. These are outbound peer events so the should be named differently.

Also I believe you should limit this query to only 100 rows since that's the limit that the API imposes.